content/community.html

```diff
@@ -1,59 +1,61 @@
 ---
 title: Community
 ---
-<div class="col-md-8 col-md-offset-2 doc-content">
-  <h1>Community</h1>
-  <p>
-    Prometheus is developed in the open. Here are some of the channels we use
-    to communicate and contribute:
-  </p>
-  <p>
-    <strong>IRC:</strong>
-	<a href="https://webchat.freenode.net/?channels=#prometheus"><code>#prometheus</code></a> on <a href="http://freenode.net/">irc.freenode.net</a>
-  </p>
-  <p>
-    <strong>Mailing list:</strong>
-    <a href="https://groups.google.com/forum/#!forum/prometheus-developers">prometheus-developers</a> Google Group
-  </p>
-  <p>
-    <strong>Twitter:</strong>
-    <a href="https://twitter.com/PrometheusIO">@PrometheusIO</a>
-  </p>
-  <p>
-    <strong>Issue tracker:</strong> We use the GitHub issue tracker for the various <a href="http://github.com/prometheus">Prometheus repositories</a>
-  </p>
-  <h1>Contributing</h1>
-  <p>
-    We welcome community contributions! Please see the
-    <code>CONTRIBUTING.md</code> file in the respective Prometheus repository
-    for instructions on how to submit changes. If you are planning on making
-    more elaborate or controversial changes, please discuss them on the mailing
-    list before sending a pull request.
-  </p>
-  <h1>Commercial support</h1>
-  <p>
-    This is a list of third-party companies which provide support or
-    consulting services for Prometheus. Prometheus is an independent open source
-    project which does not endorse any company.
-  </p>
-  <ul>
-    <li><a href="http://www.robustperception.io">Robust Perception</a></li>
-  </ul>
-  <h1>Acknowledgements</h1>
-  <p>
-    Prometheus was initially started by
-    <a href="http://www.matttproud.com">Matt T. Proud</a> and
-    <a href="http://juliusv.com">Julius Volz</a>. The majority of its
-    development has been sponsored by <a href="https://soundcloud.com">SoundCloud</a>.
-  </p>
-  <p>
-    We would also like to acknowledge early contributions by engineers from
-    <a href="https://www.docker.com/">Docker</a> and <a href="http://www.boxever.com/">Boxever</a>.
-  </p>
-  <p>
-    Special thanks to <a href="https://digitalocean.com/">DigitalOcean</a> for providing hosting resources.
-  </p>
-  <p>
-    The Prometheus logo was contributed by <a href="http://www.makingstuffmove.com/">Robin Greenwood</a>.
-  </p>
+<div class="row">
+  <div class="col-md-8 col-md-offset-2 doc-content">
```

Comment:
I can't see the issue. There are 12 columns, so with an offset of 2 the 8 columns will be centered. This appears to be the case for me.